src/test/java/uk/gov/hmcts/befta/player/BackEndFunctionalTestScenarioContextTest.java

```diff
@@ -42,7 +39,7 @@
 
     public static final String DEFINITION_STORE_HOST_KEY = "DEFINITION_STORE_HOST";
 	public static final String DEFINITION_STORE_HOST_VALUE = "http://127.0.0.1:8089/";
-	public static final String IDAM_URL_KEY = "IDAM_URL";
+    public static final String IDAM_URL_KEY = "IDAM_API_URL_BASE";
 	public static final String IDAM_URL_VALUE = "IDAM_URL_VALUE";
 	public static final String S2S_URL_KEY = "S2S_URL";
```

Comment:
Should `S2S_URL` be `S2S_URL_BASE` after rename?

Reply:
it's the test data value, which I accept is inherently misleading ad first glance.